minicluster/src/main/java/alluxio/multi/process/MultiProcessCluster.java

```diff
@@ -162,16 +162,16 @@ public synchronized void start() throws Exception {
     mState = State.STARTED;
 
     mMasterAddresses = generateMasterAddresses(mNumMasters);
+    MasterNetAddress masterAddress = mMasterAddresses.get(0);
     LOG.info("Master addresses: {}", mMasterAddresses);
     switch (mDeployMode) {
-      case NON_HA:
-        MasterNetAddress masterAddress = mMasterAddresses.get(0);
+      case UFS_NON_HA:
```

Comment:
why do we need to change the deploy types?

Reply:
because `EMBEDDED` include the `EMBEDDED_NON_HA` and `EMBEDDED_HA` situation. `NON_HA` deploy mode should include the `EMBEDDED_NON_HA` and `UFS_NON_HA` cases